File: Modul-4/Python/PRAK404-2310817320007-ZahraNabila.py
while True:
    print("Pilih Program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")

    Pilihan = int(input("Masukkan Pilihan       : "))

    if 1 <= Pilihan <= 4:
        NilaiA = float(input("Masukkan Nilai Pertama : "))
        NilaiB = float(input("Masukkan Nilai Kedua   : "))

        if Pilihan == 1:
            print(f"Hasil Penjumlahan antara {NilaiA:.2f} dengan {NilaiB:.2f} adalah {NilaiA + NilaiB:.2f}")
        elif Pilihan == 2:
            print(f"Hasil Pengurangan antara {NilaiA:.2f} dengan {NilaiB:.2f} adalah {NilaiA - NilaiB:.2f}")
        elif Pilihan == 3:
            print(f"Hasil Perkalian antara {NilaiA:.2f} dengan {NilaiB:.2f} adalah {NilaiA * NilaiB:.2f}")
        elif Pilihan == 4:
            if NilaiB != 0:
                print(f"Hasil Pembagian antara {NilaiA:.2f} dengan {NilaiB:.2f} adalah {NilaiA / NilaiB:.2f}")
            else:
                print("ERROR : Tidak dapat membagi dengan 0")

    elif Pilihan == 5:
        print("Terimakasih, telah menggunakan kalkulator Zahra Nabila")
        break
    else:
        print("Input anda salah, silahkan coba lagi")

File: Modul-4/C/PRAK404-2310817320007-ZahraNabila.c
#include <stdio.h>

int main()
{
   int Pilihan;
   float NilaiA, NilaiB;

  while (1)
  {
    printf("Pilih Program\n");
    printf("1. Penjumlahan\n");
    printf("2. Pengurangan\n");
    printf("3. Perkalian\n");
    printf("4. Pembagian\n");
    printf("5. Exit\n");

    printf("Masukkan Pilihan       : ");
    scanf("%d", &Pilihan);

    if (Pilihan > 0 && Pilihan < 5)
    {
      printf("Masukkan Nilai Pertama : ");
      scanf("%f", &NilaiA);

      printf("Masukkan Nilai Kedua   : ");
      scanf("%f", &NilaiB);

      switch (Pilihan)
      {
      case 1:
        printf("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n", NilaiA, NilaiB, NilaiA + NilaiB);
        break;
      case 2:
        printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n", NilaiA, NilaiB, NilaiA - NilaiB);
        break;
      case 3:
        printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n", NilaiA, NilaiB, NilaiA * NilaiB);
        break;
      case 4:
        if (NilaiB != 0)
        {
          printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n", NilaiA, NilaiB, NilaiA / NilaiB);
        }
        else
        {
          printf("ERROR : Tidak dapat membagi dengan 0\n");
        }
        break;
      }
    }
    else if (Pilihan == 5)
    {
      printf("Terimakasih, telah menggunakan kalkulator Zahra Nabila\n");
      break;
    }
    else
    {
      printf("Input anda salah, silahkan coba lagi\n");
    }
  }
    return 0;
}